db/maintenance/list.go

```diff
@@ -0,0 +1,56 @@
+package maintenance
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/cli/utils"
+	"github.com/Scalingo/go-scalingo/v6"
+	"github.com/Scalingo/go-utils/errors/v2"
+)
+
+func List(ctx context.Context, app string, addonName string, paginationOpts scalingo.PaginationOpts) error {
+	caser := cases.Title(language.English)
+	c, err := config.ScalingoClient(ctx)
+	if err != nil {
+		return errors.Notef(ctx, err, "get Scalingo client")
+	}
+
+	response, err := c.DatabaseListMaintenance(ctx, app, addonName, paginationOpts)
+	if err != nil {
+		return errors.Notef(ctx, err, "list the database maintenance")
+	}
+
+	t := tablewriter.NewWriter(os.Stdout)
+	t.SetHeader([]string{"ID", "Type", "Started At", "Ended At", "Status"})
+
+	for _, maintenance := range response.Maintenance {
+		startedAt := "Not started"
+		if maintenance.StartedAt != nil {
+			startedAt = maintenance.StartedAt.Local().Format(utils.TimeFormat)
+		}
+
+		endedAt := ""
+		if maintenance.EndedAt != nil {
+			endedAt = maintenance.EndedAt.Local().Format(utils.TimeFormat)
+		}
+
+		t.Append([]string{
+			maintenance.ID,
+			caser.String(string(maintenance.Type)),
+			startedAt,
+			endedAt,
+			caser.String(string(maintenance.Status)),
```

Comment:
I didn't know the `Title` method, thanks for sharing :+1: 

But I don't think that we want to format these fields (i.e. `Type`, `Status`). Becuase IMO they should be formatted like they are in platform side in case of issue in user side. That would simplify our investigation, WDYT?